polling_stations/apps/addressbase/management/commands/import_onsud.py

```diff
@@ -12,6 +12,8 @@
 """
 class Command(BaseCommand):
 
+    table_name = 'addressbase_onsud'
```

Comment:
See the comment about this pattern with the address model.

Reply:
:+1: